bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,22 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
-python:
-  - 2.7
+  - eggs
```

Comment:
Note: the former travis.cfg used the ``buildout.plonetest`` which downloads the Unified Installer. So caching downloads has there a large effect. 

If you do not use this any more the impact is smaller to zero. 

But it happens one later extends its buildout by something that downloads. I'd like the template offering this feature by default, and costs are almost zero.